include/cmdlib/Utils.hpp

```diff
@@ -0,0 +1,63 @@
+/**
+ * @file Utils.hpp cmdlib utilities
+ *
+ * This is part of the DUNE DAQ Application Framework, copyright 2020.
+ * Licensing/copyright details are in the COPYING file that you should have
+ * received with this code.
+ */
+#ifndef CMDLIB_INCLUDE_CMDLIB_UTILS_HPP_ 
+#define CMDLIB_INCLUDE_CMDLIB_UTILS_HPP_ 
+
+#include <string>
+#include <map>
+
+namespace dunedaq {
+
+    namespace url {
```

Comment:
I'm not sure that I would pull in a different global namespace than cmdlib in the cmdlib package.